karabo_bridge/simulation.py

```diff
@@ -152,22 +163,11 @@ class AGIPDModule(Detector):
         [11, 7],
     ])
 
-    const_fields = {
-        'checksum': b'\x00'*16,
-        'data': b'\x01'*416,
-        'dataId': 0,
-        'linkId': 18446744069414584335,
-        'magicNumberBegin': b'\xce\xfa\xef\xbeFDTX',
-        'magicNumberEnd': b'\xcd\xab\xad\xdeFDTX',
-        'majorTrainFormatVersion': 2,
-        'minorTrainFormatVersion': 1,
-        'reserved': b'\x00'*16,
-        'status': 0,
-    }
```

Comment:
Are these really all gone from the data the bridge forwards?

Reply:
It is not part of any of the recordings I did. Do you want me to check the karabo device?

Reply:
Maybe we can just check with @tmichela whether it's expected that those fields have gone. Or possibly they were based on what we stream from files rather than the 'real' bridge.

Reply:
@tmichela has confirmed that that